hack/verify-manifests.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_ROOT=$(dirname "${BASH_SOURCE}")/..
```

Comment:
```suggestion
SCRIPT_ROOT="$(dirname "${BASH_SOURCE}")/.."
```

otherwise it will break if the dirname includes a blank character. Also, normally tools like `shellcheck` fail if variable expansion happens without surrounding quotes.